src/Repository/HealthCheckConfigRepository.php

```diff
@@ -0,0 +1,156 @@
+     * @throws \Doctrine\DBAL\DBALException
+     */
+    public function getLastId()
+    {
+        $query = "SELECT MAX(id_health_check_config) as id FROM `{$this->dbPrefix}health_check_config`";
+
+        $statement = $this->connection->executeQuery($query);
+        if (0 === (int) $statement->errorCode()) {
+            return $statement->fetchColumn(0);
+        }
+    }
+
+    /**
+     * @throws DatabaseException
+     */
+    public function update(array $data)
+    {
+        $qb = $this->connection->createQueryBuilder();
+        $qb
+            ->update($this->dbPrefix . 'health_check_config')
+            ->set('token', ':token')
+            ->set('ips', ':ips')
+            ->setParameters([
+                'token' => $data['token'],
+                'ips' => $data['ips'],
+            ])
+        ;
+        $this->executeQueryBuilder($qb, 'HealthCheck config error');
+    }
+
+    /**
+     * @return array
+     *
+     * @throws \Doctrine\DBAL\DBALException
+     */
+    public function createTables()
+    {
+        $engine = _MYSQL_ENGINE_;
+        $this->dropTables();
+
+        $query = "CREATE TABLE IF NOT EXISTS `{$this->dbPrefix}health_check_config`(
+    			`id_health_check_config` int(10) unsigned NOT NULL auto_increment,
+    			`token` varchar(255) default NULL,
+    			`ips` varchar(255) default NULL,
+    			PRIMARY KEY (`id_health_check_config`)
+            ) ENGINE=$engine DEFAULT CHARSET=utf8";
+
+        $statement = $this->connection->executeQuery($query);
+        if (0 != (int) $statement->errorCode()) {
+            return [
+                'key' => json_encode($statement->errorInfo()),
+                'parameters' => [],
+                'domain' => 'Admin.Modules.Notification',
+            ];
+        }
+
+        return [];
+    }
+
+    /**
+     * @return array
+     *
+     * @throws \Doctrine\DBAL\DBALException
+     */
+    public function installFixtures()
+    {
+        $query = sprintf('INSERT INTO `%shealth_check_config` (`id_health_check_config`, `token`, `ips`) VALUES
+                (1, \'%s\', \'127.0.0.1, ::1\');', $this->dbPrefix, \Tools::passwdGen(128, 'RANDOM'));
+
+        $statement = $this->connection->executeQuery($query);
+        if (0 != (int) $statement->errorCode()) {
+            return [
+                'key' => json_encode($statement->errorInfo()),
+                'parameters' => [],
+                'domain' => 'Admin.Modules.Notification',
+            ];
+        }
+
+        return [];
+    }
+
+    /**
+     * @return array
+     *
+     * @throws \Doctrine\DBAL\DBALException
+     */
+    public function dropTables()
+    {
+        $sql = 'DROP TABLE IF EXISTS ' . $this->dbPrefix . 'health_check_config';
+        $statement = $this->connection->executeQuery($sql);
+        if ($statement instanceof Statement && 0 != (int) $statement->errorCode()) {
+            return [
+                'key' => json_encode($statement->errorInfo()),
```

Comment:
Is it really needed to use json_encode here? It looks like it's a message, the key is a string :thinking:

Reply:
Driver\Statement phpdoc says 

/**
     * Fetches extended error information associated with the last operation on the statement handle.
     *
     * @return mixed[] The error info array.
     */
    public function errorInfo();

Reply:
If it's an array which will be encoded in json, what is the interest of trying to translate it?